addons/main/CfgVehicles.hpp

```diff
@@ -7,13 +7,26 @@ class CfgVehicles {
         scope = 2;
         scopeCurator = 2;
         displayName = "Armor Plate";
-        author = "diawko";
+        author = "diwako";
         vehicleClass = "Items";
         class TransportItems {
             MACRO_ADDITEM(plate,1);
         };
     };
 
+    class GVAR(autoInjector): Item_Base_F {
+        scope = 2;
+        scopeCurator = 2;
+        model = "a3\Props_F_Orange\Humanitarian\Garbage\MedicalGarbage_01_Injector_F"; // model = QPATHTOF(data\autoInjector.p3d);
+        editorPreview = "\A3\EditorPreviews_F_Orange\Data\CfgVehicles\MedicalGarbage_01_Injector_F.jpg";
+        displayName = "Auto-Injector";
+        author = "alien314";
+        vehicleClass = "Items";
+        class TransportItems {
+            MACRO_ADDITEM(injector,1);
```

Comment:
You sure this is working? the CfgWeapon entry is called '`GVAR(autoInjector)`

Reply:
Probably wasn't, though I'm not sure what these CfgVehicles entries do, placing the items in editor just disappears them on mission start.